samples/BuildMinimalMaterialAppCompat/BuildMinimalMaterialAppCompat/BuildMinimalMaterialAppCompat.csproj

```diff
@@ -65,6 +66,124 @@
     <Reference Include="System.Xml" />
     <Reference Include="System.Core" />
     <Reference Include="Mono.Android" />
+    <Reference Include="Xamarin.AndroidX.Annotation.Experimental">
+      <HintPath>..\packages\Xamarin.AndroidX.Annotation.Experimental.1.2.0.2\lib\monoandroid12.0\Xamarin.AndroidX.Annotation.Experimental.dll</HintPath>
```

Comment:
Material projects are there for ages.

Reply:
> Adding more places where we explicitly reference specific assembly versions seems like what we are trying to avoid

I started this PR to reduce referencing packages, assemblies and projects all over the place and I really don't know how this was added. Maybe automatic while I was trying to make ld projects to build. Not sure. I removed it.